### DISCUSSION TITLE
Is poetry2nix abandoned?

### DISCUSSION THREAD

**Discussion by Atry** [nix-community/infra#1240] [Q&A] [answered]
There are a lot of Pull Requests but no one reviews them or merges them since 2 months ago.
https://github.com/nix-community/poetry2nix/pulls

Anything happened to the maintainers?

**Comment by Mic92** [comment]
Could be that @adisbladis moved on to https://github.com/nix-community/pyproject.nix ?
However I think this probably still have a lot of users, so it might be good to appoint a new maintainer for it. Any potential candidates?

**Comment by Ma27** [comment]
fwiw currently evaluating this for $work. If it turns out to be useful, I'd be interested in helping out with maintenance.

**Reply by Mic92** [reply]
@Ma27 did anything came out of your evaluation?

**Reply by Ma27** [reply]
@Mic92 didn't manage to finish it, too many other loose ends, sorry! Will be on vacation now and hopefully manage to go on with this after that.

**Comment by Atry** [comment, marked as answer]
@cpcloud reviewed and merged many PRs recently

**Reply by Mic92** [reply]
Yes. I set him as a maintainer for now. If @adisbladis comes back, we can discuss what to do.

**Reply by adisbladis** [reply]
While I'm not planning to be as active maintaining poetry2nix as I used to be going forward I still want to be listed as a maintainer. It's unlikely that other people will touch much of the core of poetry2nix which is where I plan to focus any maintenance efforts I spend (as I alluded to yesterday in the poetry2nix chat).
I've added myself back as a maintainer, together with @cpcloud whom I have 100% faith in.
